docs/mindsdb_sql/agents/agent.mdx

```diff
@@ -96,6 +96,19 @@ An agent can be created, deleted, queried, and updated. Here is how you can do t
        skills = ['test_skill']; -- this must be created with CREATE SKILL
     ```
 
+    Alternatively, you can create the model at the agent creation time based on [the model providers defined here](/integrations/ai-engines/langchain#openai).
```

Comment:
Here is the model is not created, only an angent. 
How it works: mindsdb agent initializes langchain with llm settings (provider, api_key, model). Creation of model in mindsdb is not required. 
The model in mindsdb is used when it exists (and provider is mindsdb)

Reply:
I'll change it to the below:
```
Alternatively, you can create an agent and define the model to be used by an agent at the agent creation time based on [the model providers defined here](/integrations/ai-engines/langchain#openai).
```